feature/experimental/p4rt/otg_tests/base_p4rt/base_p4rt_test.go

```diff
@@ -278,18 +277,16 @@ func setupP4RTClient(ctx context.Context, args *testArgs) error {
 }
 
 // Function to compare and check if the expected table is present in RPC ReadResponse
-func verifyReadReceiveMatch(expected_update []*p4_v1.Update, received_entry *p4_v1.ReadResponse) error {
-
+func verifyReadReceiveMatch(t *testing.T, expected_table *p4_v1.Update, received_entry *p4_v1.ReadResponse) error {
 	matches := 0
 	for _, table := range received_entry.Entities {
-		if diff := cmp.Diff(expected_update[0].Entity.Entity, table.Entity, protocmp.Transform()); diff != "" {
-			glog.Errorf("Table entry diff (-want +got): \n%s", diff)
-			continue
+		if cmp.Equal(table, expected_table.Entity, protocmp.Transform(), protocmp.IgnoreFields(&p4_v1.TableEntry{}, "meter_config", "counter_data")) {
+			t.Logf("Table match succesful")
```

Comment:
I'm not sure this log statement provides anything useful in the log. Consider dropping

Reply:
Taken care of this.